apps/sonda/sonda.py

```diff
@@ -179,21 +181,20 @@ def main():
   log_with_utc(f'Store nodes to query: {store_nodes}')
 
   # Start Prometheus HTTP server at port 8004
-  start_http_server(8004)
+  start_http_server(args.metrics_port)
```

Comment:
Maybe adapt the previous comment too?

Reply:
I am not sure what you mean by this?

Reply:
Oh you mean to adapt the comment in the script, sure